packages/terra-icon/scripts/src/generate-example/generateStaticIcons.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable */
 import fs from 'fs';
 
-const outputfile = fs.createWriteStream('../../packages/terra-site/src/examples/icon/IconThemeable.jsx', { flags: 'w' });
+const outputfile = fs.createWriteStream('../../packages/terra-site/src/examples/icon/IconStatic.jsx', { flags: 'w' });
 
 const generateIconAll = iconObjs => new Promise((resolve) => {
 
   const staticIcons = iconObjs.filter(function(iconObj){
```

Comment:
being [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) just expects a boolean if the result should be filtered, this can be simplified to 
```js
const staticIcons = iconObjs.filter(icon => !icon.themeable);
```